setup.py

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env python
+
+# Copyright Least Authority Enterprises.
+# See LICENSE for details.
+
+import setuptools
+
+_metadata = {}
+with open("src/txkube/_metadata.py") as f:
+    exec(f.read(), _metadata)
+
+setuptools.setup(
+    name="txkube",
+    version=_metadata["version_string"],
+    description="A Twisted-based Kubernetes client.",
+    author="txkube Developers",
+    url="https://github.com/leastauthority.com/txkube",
+    license="MIT",
+    package_dir={"": "src"},
+    packages=setuptools.find_packages(where="src"),
+    install_requires=[
+        "twisted[tls]",
```

Comment:
Weird :/  My only guess is there was a transient network error while trying to find pyasn1?  The PyPI project looks okay and I can download the wheel now... Apart from a random obscure pip bug I don't have any other guesses.